rdr_service/offline/export_va_workqueue.py

```diff
@@ -182,7 +182,7 @@ def _get_json_fields(participant):
 
 def _export_datetime(api_datetime):
     if api_datetime:
-        return datetime.datetime.fromisoformat(api_datetime).strftime("%m/%d/%Y %H:%M:%S")
+        return datetime.datetime.fromisoformat(api_datetime.replace("Z", "")).strftime("%m/%d/%Y %H:%M:%S")
```

Comment:
This works, an alternative method is using the parse() or isoparse() methods from the `dateutil` library. Something like 
`from dateutil.parser import parse`
`return parse(api_datetime).strftime("%m/%d/%Y %H:%M:%S")`